ct.yaml

```diff
@@ -1,7 +1,5 @@
 # See https://github.com/helm/chart-testing#configuration
-remote: origin
 chart-dirs:
   - charts
-chart-repos:
-  - incubator=https://kubernetes-charts-incubator.storage.googleapis.com
-helm-extra-args: --timeout=500s
```

Comment:
Since this value is not backward-compatible with helm/chart-testing-action@v1.0.0-rc.2 and helm/chart-testing-action@v1.0.0-alpha.3, ref to #85, remove it.

Reply:
There is a workaround way to inject different default value to each job, ref to https://github.com/helm/chart-testing-action/issues/36#issuecomment-655076576